docs/guides/developer/docs/architecture/api/playlists-api.md

```diff
@@ -0,0 +1,215 @@
+[TOC]
+
+# Information
+
+The Playlists API is available since API version 1.11.0.
+
+### GET /api/playlists/playlist.json
+
+Returns a playlist.
+
+__Response__
+
+`200 (OK)`: A playlist as JSON.
+
+| Field              | Type                        | Description                                               |
+|--------------------|-----------------------------|-----------------------------------------------------------|
+| `id`               | [`long`](types.md#basic)    | The playlist identifier                                   |
+
+
+__Example__
+
+```json
```

Comment:
No idea where to best put this, but I think the docs should contain the following information somewhere:
- the `updated` field is in ms since the epoch
- What the deal with the `type` field is. Explain what happens when videos inside a playlist are not accessible.

Reply:
Should probably go into `types.md`, no?